File: ex_answer/ex1.py
class Address:
    def __init__(self, country, province, city, street, zip_code):
        self.country = country
        self.province = province
        self.city = city
        self.street = street
        self.zip_code = zip_code

    def returnAddress(self):
        return self.country + ' ' +  self.province + ' ' + self.city + ' ' +  self.street + ' '  + str(self.zip_code)



add = Address('China', 'Jiangxi', 'Nanchang', 'xxxRoad', 330000)
print(add.returnAddress())

File: ex_answer/ex1.java
class Address{
    String country;
    String province;
    String city;
    String street;
    String zip_code;

    public Address(String country, String province, String city, String street, String zip_code){
        this.country = country;
        this.province = province;
        this.city = city;
        this.street = street;
        this.zip_code = zip_code;
    }

    public String returnAddress(){
        return this.country + " " + this.province + " " + this.city + " " + this.street + " " + this.zip_code;
    }
}

public class ex1 {
    public static void main(String args[]){
        Address add = new Address("China", "Jiangxi", "Nanchang", "xxxRoad", "330000");
        System.out.println(add.returnAddress());
    }    
}
